lib/lightning_web/live/project_live/settings.html.heex

```diff
@@ -0,0 +1,112 @@
+<Layout.page_content>
+  <:header>
+    <Layout.header socket={@socket}>
+      <:title><%= @page_title %></:title>
+    </Layout.header>
+  </:header>
+  <Layout.centered>
+    <div class=" relative flex">
+      <div class="pr-4 pt-0 sticky top-0 self-start rounded-md">
+        <div class="flex-none">
+          <LightningWeb.Components.Common.tab_bar
+            orientation="vertical"
+            id={@project.id}
+            default_hash="settings"
+          >
+            <LightningWeb.Components.Common.tab_item
+              orientation="vertical"
+              hash="settings"
```

Comment:
I think the default hash should be `project` no? and the project tab should have a `project` hash? Right?

I mean I guess I see why it's called `settings` since it's the project settings..
But imo the uri looks weird with `.../settings#settings`.

Not super strong opinions on this.. your thoughts?

Reply:
I agree. This is done, thanks.